brokers/che-plugin-broker/cmd/config.json

```diff
@@ -1,32 +0,0 @@
-[
-{
-    "id": "che-service-plugin",
-    "version": "0.0.1",
-    "type": "Che plugin",
-    "name": "Che Samples REST API Sidecar Plugin",
-    "title": "Che Samples REST API Sidecar Plugin",
-    "description": "Che Plug-in with Theia plug-in and container definition providing a service",
-    "icon": "https://www.eclipse.org/che/images/ico/16x16.png",
-    "url": "https://github.com/ws-skeleton/che-service-plugin/releases/download/untagged-5c4c888ff2de8ae7a5e2/che-service-plugin.tar.gz"
-},
-{
-    "id": "org.eclipse.che.editor.theia",
-    "version": "1.0.0",
-    "type": "Che Editor",
```

Comment:
Was it done by a purpose that this config contains two Che Theia editors?

Reply:
Not really. It checked that it is possible to use either tar.gz or raw link to YAML